GitHub Action Failure

Dependabot is suggesting that I bump setup-surreal from v1 to v2, but the latter fails.

```
Run surrealdb/setup-surreal@v2
  with:
    surrealdb_version: latest
    surrealdb_port: 8000
    surrealdb_username: ***
    surrealdb_password: ***
    surrealdb_log: debug
    surrealdb_additional_args: --allow-all
    surrealdb_auth: false
    surrealdb_strict: false
    surrealdb_retry_count: 30
Run SURREALDB_VERSION="latest"
  
  SURREALDB_VERSION="latest"
  
  # Download the SurrealDB binary when the version is not latest
  if [ "$SURREALDB_VERSION" = "latest" ]; then
    curl --proto '=https' --tlsv1.2 -sSf https://install.surrealdb.com/ | sh
  else
    curl --proto '=https' --tlsv1.2 -sSf https://install.surrealdb.com/ | sh -s -- --version latest
  fi
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}

 .d8888b.                                             888 8888888b.  888888b.
d88P  Y88b                                            888 888  'Y88b 888  '88b
Y88b.                                                 888 888    888 888  .88P
 'Y888b.   888  888 888d888 888d888  .d88b.   8888b.  888 888    888 8888888K.
    'Y88b. 888  888 888P'   888P'   d8P  Y8b     '88b 888 888    888 888  'Y88b
      '888 888  888 888     888     88888888 .d888888 888 888    888 888    888
Y88b  d88P Y88b 888 888     888     Y8b.     888  888 888 888  .d88P 888   d88P
 'Y8888P'   'Y88888 888     888      'Y8888  'Y888888 888 8888888P'  8888888P'

Fetching the latest database version...
Fetching the host system architecture...
Installing surreal-v2.0.4 for linux-amd64...

SurrealDB successfully installed in:
  /usr/local/bin/surreal

To see the command-line options run:
  surreal help
To start an in-memory database server run:
  surreal start --log debug --user *** --pass *** memory
For help with getting started visit:
  https://surrealdb.com/docs

Run SURREALDB_VERSION="latest"
  
  SURREALDB_VERSION="latest"
  SURREALDB_PORT="8000"
  SURREALDB_USERNAME="***"
  SURREALDB_PASSWORD="***"
  SURREALDB_AUTH="false"
  SURREALDB_STRICT="false"
  SURREALDB_LOG="debug"
  SURREALDB_ADDITIONAL_ARGS="--allow-all"
  SURREALDB_ALLOW_ALL=""
  
  if [ -z "$SURREALDB_USERNAME" ]; then
    SURREALDB_USERNAME="-u ***"
  else
    SURREALDB_USERNAME="-u $SURREALDB_USERNAME"
  fi
  
  if [ -z "$SURREALDB_PASSWORD" ]; then
    SURREALDB_PASSWORD="-p ***"
  else
    SURREALDB_PASSWORD="-p $SURREALDB_PASSWORD"
  fi
  
  if [[ "$SURREALDB_VERSION" = v1.* ]]; then
    if [ "$SURREALDB_AUTH" == "true" ]; then
      SURREALDB_AUTH="--auth"
    else
      SURREALDB_AUTH=""
    fi
  else
    if [ "$SURREALDB_AUTH" = "true" ]; then
      SURREALDB_AUTH=""
    else
      SURREALDB_AUTH="--unauthenticated"
    fi
  fi
  
  if [[ -z "$SURREALDB_PORT" ]]; then
    SURREALDB_PORT="--bind 0.0.0.0:8000"
  else
    SURREALDB_PORT="--bind 0.0.0.0:$SURREALDB_PORT"
  fi
  
  if [ "$SURREALDB_STRICT" = "true" ]; then
    SURREALDB_STRICT="--strict"
  fi
  
  if [ -z "$SURREALDB_LOG" ]; then
    SURREALDB_LOG="--log trace"
  else
    SURREALDB_LOG="--log $SURREALDB_LOG"
  fi
  
  if [ -z "$SURREALDB_ADDITIONAL_ARGS" ]; then
    SURREALDB_ADDITIONAL_ARGS=""
  fi
  
  cd /usr/local/bin
  
  nohup surreal start \
    $SURREALDB_USERNAME \
    $SURREALDB_PASSWORD \
    $SURREALDB_AUTH \
    $SURREALDB_PORT \
    $SURREALDB_STRICT \
    $SURREALDB_LOG \
    $SURREALDB_ADDITIONAL_ARGS &
    
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
error: invalid value 'false' for '[PATH]': Provide a valid database path parameter

For more information, try '--help'.
Run URL="http://localhost:8000/health"
  URL="http://localhost:8000/health"
  MAX_ATTEMPTS="30"
  ATTEMPT=0
  
  while [ $ATTEMPT -lt $MAX_ATTEMPTS ]; do
      RESPONSE=$(curl -s -o /dev/null -w "%{http_code}" $URL)
      
      if [ "$RESPONSE" -eq 200 ]; then
          echo "SurrealDB instance is up and running, continuing..."
          break
      fi
      
      ATTEMPT=$((ATTEMPT + 1))
      sleep 1
  done
  
  if [ "$ATTEMPT" -ge "$MAX_ATTEMPTS" ]; then
      echo "SurrealDB instance could not be contacted, aborting."
      exit 1
  fi
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
Error: Process completed with exit code 7.
```

--- a/action.yml
+++ b/action.yml
@@ -104,6 +104,8 @@ runs:
 
         if [ "$SURREALDB_STRICT" = "true" ]; then
           SURREALDB_STRICT="--strict"
+        else
+          SURREALDB_STRICT=""
         fi
 
         if [ -z "$SURREALDB_LOG" ]; then

